### DISCUSSION TITLE
Where can I check the differences of a new release?

### DISCUSSION THREAD

**Discussion by marsch1975** [Martchus/syncthingtray#271] [General]
Hi,

where can I check the differences of a new release?

Cheers
Markus

**Comment by Martchus** [comment]
You can compare the difference directly on GitHub between any two release tags, e.g. this link will compare between v1.5.0 and v1.5.4: https://github.com/Martchus/syncthingtray/compare/v1.5.0...v1.5.4

Of course you can also just clone the repo and use your local Git client.

Note that other repositories such as c++utilities, qtutilities and of course upstream Qt might also have many relevant changes. (For Qt this blog often contains relevant posts: https://www.qt.io/blog/tag/loop)

**Comment by marsch1975** [comment]
Thank you.

I just want to see Bug fixes, new stuff, ... Not the code itself...

How can I change the version v1.5.0 to v1.5.3? Seams I cant change it on the top of the page...

When you chage to libsycthing version higher then I'm using, because I let tray start a lower version of syncthing, is it bad, should I really change the syncthing version aswell?

Thanks
Markus

**Reply by Martchus** [reply]
> I just want to see Bug fixes, new stuff, ... Not the code itself...

GitHub actually shows a list of commits on the page I mentioned. Of course those are rather detailed. So far I don't provide a high-level list and it is probably too much effort to do that. I sometimes post a summary with "news" in the forum thread, though. 

> When you chage to libsycthing version higher then I'm using, because I let tray start a lower version of syncthing, is it bad, should I really change the syncthing version aswell?

It is not a tested/supported configuration but it is very likely *not* going to be a problem. Syncthing has a stable API and I would normally not make use of a new APIs unconditionally.

**Comment by marsch1975** [comment]
OK, thank you.

Two more questions please. :)

When I replace the folder SyncthingTray 1.5.3 with the folder SyncThingTray 1.5.4, how can I import my settings?

Do I get it right? There is a Synthing version in every SyncthingTray version?
So I could klick on "use build in syncthing library" and don't need to download a seperate Syncthing exe?

Cheers
Markus?

**Reply by Martchus** [reply]
> When I replace the folder SyncthingTray 1.5.3 with the folder SyncThingTray 1.5.4, how can I import my settings?

The settings are stored elsewhere (see the README for details) so you don't have to import anything.

> Do I get it right? There is a Synthing version in every SyncthingTray version?
So I could klick on "use build in syncthing library" and don't need to download a seperate Syncthing exe?

Yes to all, checkout the README for details.
